charts/testkube-dashboard/values.yaml

```diff
@@ -124,6 +124,11 @@ ingress:
     #     - testkube.example.com
     #   secretName: testkube-cert-secret
 
+## Proxy prefix to use when deploying
+## to a subpath and ingress.path contains a regex
+## Sets the env variable REACT_APP_ROOT_ROUTE
+proxyPrefix: ""
```

Comment:
Could you please duplicate this into `testkube/values.yaml` file?

Reply:
Done.  I'm not sure how you want to handle Chart versions so I have not touched those.